### ISSUE TITLE
Screen readers report incorrect position/size of virtualized Command Palette menu items

### ISSUE THREAD

**Issue by jayspadie** [Microsoft/monaco-editor#444]
monaco-editor version: 0.8.3
Browser:  Any
OS: Windows 10

During accessibility testing, it was found that the virtualized lists such as that in the Command Palette (F1) will make screen readers report an incorrect total count based on the actual DOM elements currently rendered.  For example, the screenshot below of NVDA and Firefox initially includes a total count of 12 until keyboarding past the bottom item (Delete All Right).  Also, I am not sure what "Level 1" for each item refers to.

Perhaps [aria-setsize](https://www.w3.org/TR/wai-aria/states_and_properties#aria-setsize) and [aria-posinset](https://www.w3.org/TR/wai-aria/states_and_properties#aria-posinset) could be used to correct it.

![image](https://cloud.githubusercontent.com/assets/5246869/26264243/120a2e18-3caa-11e7-9629-2e44085a00ba.png)

**Comment by alexandrudima** [comment]
This might be a Firefox specific issue

**Comment by bpasero** [comment]
I was not aware of these aria rules and I think we should set them. I created https://github.com/Microsoft/vscode/issues/27900 because I think this should happen on the tree/list level.

**Comment by jayspadie** [comment]
From what I've seen, NVDA with Firefox does a "better" job of announcing "x of y" for lists of items (tabs, menu items, lists, etc.), while this is typically not announced when using Chrome.  NVDA with Firefox is generally considered to follow the ARIA specs better than with Chrome, as I understand it.  Assuming Chrome continues to improve support for screen readers, I would expect it to behave the same someday.  So, I wouldn't really consider this to be a bug with Firefox.

**Comment by alexandrudima** [comment]
fyi @chrisdias @jayspadie 

There has been an investigation and discussions regarding this in https://github.com/Microsoft/vscode/issues/27900 and it doesn't look like it will make it in this release.

**Comment by isidorn** [comment]
Closed via https://github.com/Microsoft/vscode/commit/5241c72bd3764a4c3b9072a4d34e9e10a6c7867b
